examples/E2EBert/news_classifier.py

```diff
@@ -23,6 +23,8 @@
 import torchtext.datasets as td
 from transformers import BertModel, BertTokenizer, AdamW
 
+device = torch.device("cuda:0" if torch.cuda.is_available() else "cpu")
```

Comment:
@Arvind-Ideas2IT please use cuda instead of cuda with gpu id and remove the commented code on Line#431

Reply:
@HamidShojanazeri  Thanks for pointing it out.  I have updated this.